src/app/auth/register/register.component.html

```diff
@@ -0,0 +1,26 @@
+<form class="example-form" [formGroup]="registerForm" (ngSubmit)="onSubmit()">
+  <div *ngIf="registerForm.controls['password'].invalid && (registerForm.controls['password'].dirty || registerForm.controls['password'].touched)"
```

Comment:
why using `(registerForm.controls['password'].dirty || registerForm.controls['password'].touched)` not `&&`